src/utils_general/statistics.py

```diff
@@ -109,3 +111,38 @@ def calc_mpe(observations: np.array, forecast: np.array) -> float:
         / len(observations.time)
         * 100
     )
+
+
+def calc_crps(
+    observations: xr.DataArray,
+    forecasts: xr.DataArray,
+    normalization: str = None,
+    member_dim: str = "number",
+) -> float:
+    """
+    observations and forecasts must have the same shape in all
+    dimensions except the member_dim dimension
```

Comment:
And maybe mention that only the forecast should have the ensemble member dimension